### ISSUE TITLE
Considering showing the window label in multiple corners

### ISSUE THREAD

**Issue by Wilfred** [abo-abo/ace-window#129]
I frequently have point on the first visible line in a window. As a result, it becomes difficult to read the window labels:

![ace_window](https://user-images.githubusercontent.com/70800/39296359-a02b6522-4938-11e8-9dfc-24cd19f84354.png)

It's hard to distinguish `s` from `d` here. Would it be possible to show the label in multiple corners, so at least one is always clearly visible?

**Comment by abo-abo** [comment]
> Would it be possible to show the label in multiple corners, so at least one is always clearly visible?

Yes. It's easier to show it in top-left and bottom-left. But I think overlays with a bunch of spaces can be added to show the keys also in top-right and bottom-right.
